pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -315,7 +315,7 @@ contract TimelockAuthorizer is IAuthorizer, TimelockAuthorizerManagement {
             require(msg.sender == getTimelockExecutionHelper(), "GRANT_MUST_BE_SCHEDULED");
         }
 
-        require(!_isPermissionGranted[actionId][account][where], "PERMISSION_ALREADY_GRANTED");
+        require(!hasPermission(actionId, account, where), "PERMISSION_ALREADY_GRANTED");
```

Comment:
Maybe add a comment that this is now also checking for global permission for this action, as well as specific permission at where?